engine/HAL/NATIVE/luos_hal_config.h

```diff
@@ -0,0 +1,54 @@
+/******************************************************************************
+ * @file luosHAL_Config
+ * @brief This file allow you to configure LuosHAL according to your design
+ *        this is the default configuration created by Luos team for this MCU Family
+ *        Do not modify this file if you want to ovewrite change define in you project
+ * @Family x86
+ * @author Luos
+ * @version 0.0.0
+ ******************************************************************************/
+#ifndef _LUOSHAL_CONFIG_H_
+#define _LUOSHAL_CONFIG_H_
+
+#ifndef MCUFREQ
+#define MCUFREQ 100000000 // MCU frequence
```

Comment:
not really important but how you define this?

Reply:
I any case I will remake a lot of things on those files.
This PR is far from ready....